### ISSUE TITLE
SentimentUtils: Separate data and logic

### ISSUE THREAD

**Issue by Chr1st1anSzech** [codingupastorm/vadersharp#4]
I would recommend to write all constants, words and other Information in ressource files, e.g. JSON or XML. That would support a more strict separation of data and logic. Furthermore, it simplifies i18n.

**Comment by codingupastorm** [comment]
Good idea - I wanted to first transcribe and make sure everything works but am happy to improve by moving the constants to resource files
